osbrain/tests/test_nameserver.py

```diff
@@ -342,8 +344,10 @@ def test_random_nameserver_process():
     # Raising exceptions
     with pytest.raises(ValueError):
         random_nameserver_process(port_start=-1, port_stop=-2)
-    with pytest.raises(RuntimeError):
-        random_nameserver_process(port_start=22, port_stop=22, timeout=0.5)
+    if sys.platform != 'win32':
+        with pytest.raises(RuntimeError):
+            random_nameserver_process(port_start=22,
+                                      port_stop=22, timeout=0.5)
```

Comment:
I'd say we simply skip this test on Windows for now and revert the changes.

Reply:
Makes sense.

Reply:
What about `random_nameserver_process()` then? Shouldn't we test it works?

Reply:
For now let us just skip those failing tests. We will fix them later, as separate PRs, analyzing the situation for each case.